### PR TITLE
add mapillary rules

### PR DESCRIPTION
**Pull request by MorrisJobke**
@Raydiation I added this to the owncloud news app, but the scraper doesn't fetches the full content. What did I wrong?

### CODE REVIEW

File: lib/PicoFeed/Rules/blog.mapillary.com.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?php
+return array(
+	'grabber' => array(
```
Comment (Raydiation) [inline lib/PicoFeed/Rules/blog.mapillary.com.php]: Indention is 4 spaces

**Comment by Raydiation** [comment]
Should actually work I think, maybe the content is generated with JavaScript, can you check?

**Comment by mkresin** [comment]
The rulefile works for me with miniflux. Personally, I would remove the ```//h2[@class="blog-post__title"]``` part, since miniflux displays the heading twice.

screenshot 289 traffic sign updates - mozilla firefox

@MorrisJobke Do you want to get this rule file merged or is this a (owncloud news app) support request?

**Comment by Raydiation** [comment]
Ah, did you turn on the website scraper for the website? You have to enable it by opting in, it's a separate button in the feed menu

**Comment by MorrisJobke** [comment]
> @MorrisJobke Do you want to get this rule file merged or is this a (owncloud news app) support request?

Both :P I changed the indentation and it would be nice to have it upstream, isn't it? ;)

Feel free to merge it if it fits into your library.

**Pull request merged**